src/data/reducers/filters.test.js

```diff
@@ -0,0 +1,172 @@
+import filter from './filters';
+import actions from '../actions/filters';
+import gradeActions from '../actions/grades';
+import initialFilters from '../constants/filters';
+
+const initialState = {};
+
+const expectedFilterType = 'homework';
+const expectedAssignmentId = 'assignment 1';
+const expectedAssignment = {
+  type: expectedFilterType,
+  id: expectedAssignmentId,
+};
+const testingState = {
+  assignment: expectedAssignment,
+};
+
+describe('filter reducer group', () => {
+  it('has initial state', () => {
+    expect(filter(undefined, {})).toEqual(initialState);
+  });
+
+  it('filter by assignment type', () => {
```

Comment:
These should indicate the action they are responding to, not a description of the event.

Reply:
example: 
```
describe('handle actions.update.assignmentType', () => {
  it('loads the assignmentType from the action', () => { ... });
  it('clears the assignment if the filterType is not empty and doesn't match the selected assignment', () => { ... });
}):
```